### PR TITLE
Polish Azure.ResourceManager.Subscription

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/subscription/Azure.ResourceManager.Subscription/api/Azure.ResourceManager.Subscription.netstandard2.0.cs [hunk 1]
```diff
@@ -166,177 +177,106 @@ public partial class EnabledSubscriptionId
...
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.Subscription.Models.Provisioning left, Azure.ResourceManager.Subscription.Models.Provisioning right) { throw null; }
-        public static implicit operator Azure.ResourceManager.Subscription.Models.Provisioning (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.Subscription.Models.Provisioning left, Azure.ResourceManager.Subscription.Models.Provisioning right) { throw null; }
-        public override string ToString() { throw null; }
-    }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct ProvisioningState : System.IEquatable<Azure.ResourceManager.Subscription.Models.ProvisioningState>
+    public partial class ServiceTenantResponse
```
Comment (Yao725) [inline sdk/subscription/Azure.ResourceManager.Subscription/api/Azure.ResourceManager.Subscription.netstandard2.0.cs:185]: It's better to change the suffix `Response`.

File: sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md [hunk 2]
```diff
@@ -16,9 +16,34 @@ skip-csproj: true
 modelerfour:
   flatten-payloads: false
 
+mgmt-debug: 
+  show-serialized-names: true
```
Comment (Yao725) [inline sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md:20]: Why do we need to show the serialized name in comment?

**Review by Yao725 — APPROVED** [review]

File: sdk/subscription/Azure.ResourceManager.Subscription/api/Azure.ResourceManager.Subscription.netstandard2.0.cs (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/subscription/Azure.ResourceManager.Subscription/api/Azure.ResourceManager.Subscription.netstandard2.0.cs:185]: drop the suffix

File: sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/subscription/Azure.ResourceManager.Subscription/src/autorest.md]: fixed

**Pull request merged**